packages/ag-charts-enterprise/src/features/status-bar/statusBar.ts

```diff
@@ -139,7 +157,27 @@ export class StatusBar
         const offsetTop = maxFontSize + (lineHeight - maxFontSize) / 2;
 
         let left = 0;
-        for (const { label, title, value, domain } of Object.values(this.labels)) {
+        for (const { label, title, value, domain, formatter } of Object.values(this.labels)) {
+            let maxValueWidth = 0;
+            if (domain.length > 0) {
+                domain.length > 0;
```

Comment:
SonarCloud flagging this line: https://sonarcloud.io/code?id=ag-charts-community-latest&pullRequest=1825&selected=ag-charts-community-latest%3Apackages%2Fag-charts-enterprise%2Fsrc%2Ffeatures%2Fstatus-bar%2FstatusBar.ts&line=163

Reply:
Ah my bad. I refactored a ternary and clearly left this in